src/main/java/frc/utils/devicewrappers/TalonFXWrapper.java

```diff
@@ -30,8 +30,8 @@ public void applyConfiguration(TalonFXConfiguration configuration) {
      * Performs latency compensation on signal using the signalSlope and signal's
      * latency to determine the magnitude of compensation.
      */
-    public double getLatencyCompensatedValue(StatusSignal<Double> value, StatusSignal<Double> valueSlope) {
-        return BaseStatusSignal.getLatencyCompensatedValue(value.refresh(), valueSlope.refresh());
+    private double getLatencyCompensatedValue(StatusSignal<Double> value, StatusSignal<Double> valueSlope) {
+        return BaseStatusSignal.getLatencyCompensatedValue(value, valueSlope);
```

Comment:
Maybe move it to phoenix pro utils, or delete it and just use "BaseStatusSignal.getLatencyCompensatedValue" in this class.